include/atomics/receiver_cadmium.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#ifndef __RECEIVER_CADMIUM_HPP__
+#define __RECEIVER_CADMIUM_HPP__
+
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <limits>
+#include <math.h> 
+#include <assert.h>
+#include <memory>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <chrono>
+#include <algorithm>
+#include <limits>
+#include <random>
+#include "../data_structures/message.hpp"
+
+using namespace cadmium;
+using namespace std;
+
+//Port definition
+struct Receiver_defs {
+	struct out : public out_port<Message_t> {
+    };
+    struct in : public in_port<Message_t> {
+    };
+};
+
+template<typename TIME>
+class Receiver {
+	using defs=Receiver_defs; // putting definitions in context
+	public:
+		//Parameters to be overwritten when instantiating the atomic model
```

Comment:
Check indentation please

Reply:
Please download the files to your local and check the indentations.
Its looking perfect in notepad++ or other editor with 4 spaces